### PR TITLE
enable css for StackBase, remove for StackLayout

### PR DESCRIPTION
**Pull request by Giviruk**
enable css for StackBase for issue https://github.com/dotnet/maui/issues/21843

### CODE REVIEW

**Comment by Giviruk** [comment]
This is my first PR, I will try to fix the errors as soon as possible

**Review by jsuarezruiz — COMMENTED** [review]
There are some errors and cannot pass the build:

```
D:\a\_work\1\s\src\Controls\src\Core\Properties\AssemblyInfo.cs(158,83): error CS0117: 'StackBase' does not contain a definition for 'OrientationProperty' [D:\a\_work\1\s\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net8.0-windows10.0.19041.0]
C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.5.240311000\buildTransitive\Microsoft.UI.Xaml.Markup.Compiler.interop.targets(841,9): error MSB3073: The command ""C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.5.240311000\buildTransitive\..\tools\net6.0\..\net472\XamlCompiler.exe" "D:\a\_work\1\s\artifacts\obj\Controls.Core\Release\net8.0-windows10.0.19041.0\\input.json" "D:\a\_work\1\s\artifacts\obj\Controls.Core\Release\net8.0-windows10.0.19041.0\\output.json"" exited with code 1. [D:\a\_work\1\s\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net8.0-windows10.0.19041.0]
D:\a\_work\1\s\src\Controls\src\Core\Properties\AssemblyInfo.cs(158,83): error CS0117: 'StackBase' does not contain a definition for 'OrientationProperty' [D:\a\_work\1\s\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net8.0-tizen]
D:\a\_work\1\s\src\Controls\src\Core\Properties\AssemblyInfo.cs(158,83): error CS0117: 'StackBase' does not contain a definition for 'OrientationProperty' [D:\a\_work\1\s\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net8.0-windows10.0.20348.0]
C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.5.240311000\buildTransitive\Microsoft.UI.Xaml.Markup.Compiler.interop.targets(841,9): error MSB3073: The command ""C:\Users\cloudtest\.nuget\packages\microsoft.windowsappsdk\1.5.240311000\buildTransitive\..\tools\net6.0\..\net472\XamlCompiler.exe" "D:\a\_work\1\s\artifacts\obj\Controls.Core\Release\net8.0-windows10.0.20348.0\\input.json" "D:\a\_work\1\s\artifacts\obj\Controls.Core\Release\net8.0-windows10.0.20348.0\\output.json"" exited with code 1. [D:\a\_work\1\s\src\Controls\src\Core\Controls.Core.csproj::TargetFramework=net8.0-windows10.0.20348.0]
    11 Error(s)
```

Let me know if can help with something :)

**Comment by Giviruk** [comment]
I looked and saw that the vertical layout and horizontal layout do not have OrientationProperty, so I returned OrientationProperty for StackLayout and left SpacingProperty for StackBase

File: src/Controls/src/Core/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -147,15 +147,16 @@
...
 [assembly: StyleProperty("-maui-max-track-color", typeof(Slider), nameof(Slider.MaximumTrackColorProperty))]
 [assembly: StyleProperty("-maui-thumb-color", typeof(Slider), nameof(Slider.ThumbColorProperty))]
-[assembly: StyleProperty("-maui-spacing", typeof(StackLayout), nameof(StackLayout.SpacingProperty))]
-[assembly: StyleProperty("-maui-orientation", typeof(StackLayout), nameof(StackLayout.OrientationProperty))]
 
 // TODO ezhart 2021-07-16 When we fix #1634, we'll need to enable this so the CSS applies 
 //[assembly: StyleProperty("-maui-spacing", typeof(StackLayout), nameof(StackLayout.SpacingProperty))]
 
 // TODO ezhart 2021-07-16 When we create the new composed StackLayout, we'll need to ensure we have this enabled 
 //[assembly: StyleProperty("-maui-orientation", typeof(StackLayout), nameof(StackLayout.OrientationProperty))]
```
Comment (mattleibow) [inline src/Controls/src/Core/Properties/AssemblyInfo.cs:155]: Can we also remove these comments?

**Review by mattleibow — APPROVED** [review]

**Comment by Giviruk** [comment]
can someone explain this problem with the code or something else?

**Comment by mattleibow** [comment]
something weird happened and navigation stopped working in just 1 test. Retrying.

**Pull request merged**